ts/store/reducers/wallet/index.ts

```diff
@@ -20,14 +23,16 @@ export type WalletState = Readonly<{
   payment: PaymentState;
   pspsById: PspStateById;
   abi: AbiState;
+  bancomat: BancomatState;
```

Comment:
Can we change the store path from `wallet/bancomat/onboarding/pans` to `wallet/onboarding/bancomat/pans` just to follow the project foldering?